apps/docs/content/components/VideoPlayer.mdx

```diff
@@ -14,30 +14,35 @@ import {VideoPlayer} from '@primer/react-brand'
 
 ## Examples
 
+{/* Note: These assets will not load in local development. The way gatsby works prefixes the assets URLs which cannot currently be done in these files. */}
```

Comment:
This renders actual markup in the docs 👇 , can you use a valid markdown comment here instead please?

![Screenshot 2023-08-29 at 09 34 45](https://github.com/primer/brand/assets/13340707/79b7458f-b3dd-4e97-9e37-1c459f9eff5b)

Reply:
This has been updated in the following [SHA](https://github.com/primer/brand/pull/407/commits/4336fa1b489463d256e5d2e3aab7c83cdc0c821d) I will test the build once it renders. @rezrah what are the steps to make sure this affects prod correctly?

Reply:
> @rezrah what are the steps to make sure this affects prod correctly? 

Once merged, the video player docs examples should start working.

There's no way to test this on preview urls, so you can either test test in prod **or** you locally by building the prod docs and serving output on a local web serving with a `basePath` of `brand`, to simulate the prod enironment. That's how I've typically done it before.